TTC.Deployment.Tests/RoleHelper.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using Amazon.IdentityManagement.Model;
+using Amazon.IdentityManagement;
+using Amazon.Runtime;
+
+namespace TTC.Deployment.Tests
+{
+    internal class RoleHelper
```

Comment:
So - I'm not mad keen on this helper.

Perhaps a user factory would be better?

GiveMeAUser.WhoCanAssumeS3Role
GiveMeAUser.WhoCanAssumeGodRole

Or something else - not sure.  Only thing I know is its intent currently isn't that clear